src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -49,9 +42,8 @@ class AccountControllerTest @Autowired constructor(
         )
     )
 
-    @Nested
+    @EndpointTest
     @DisplayName("GET /accounts")
```

Comment:
I spent some time trying to merge these two annotations but didn't succeed. Some help would be appreciated, you can see the solution I tried in the thread I opened: https://stackoverflow.com/questions/75077738/kotlin-aliasfor-not-overriding-attribute-from-displayname

Reply:
I spent some time looking at this myself and it seems to me that your solution is correct and does work when the composed annotation attribute is not present: https://chrysanthium.com/spring-annotation-composition
However, `DisplayName` has no default value for its `value` field, which may be causing the issue. Not sure why this works this way but found a possibly similar case: https://stackoverflow.com/questions/69280914/spring-aliasfor-not-working-for-profile-annotation